docs/devel/revive-hf.rst

```diff
@@ -0,0 +1,20 @@
+devel/revive-hf
+===============
+
```

Comment:
remember to add a `dfhack-tool` directive here. See https://docs.dfhack.org/en/stable/docs/Documentation.html#header-format for details. Is this script specifically intended for adventure mode?